test/lib/org.test.ts

```diff
@@ -78,7 +78,7 @@ describe('listProjects', () => {
       },
       projects: expect.any(Array),
     });
-    expect(res.projects[0]).toMatchObject({
+    expect(res.data[0]).toMatchObject({
```

Comment:
why did this need to change? The function should still be returning all the same, but the API we call is different